ebuild-writing/common-mistakes/text.xml

```diff
@@ -477,8 +491,42 @@ at least when the ebuild hits the stable branch.
 </p>
 </body>
 </subsection>
-</section>
 
+<subsection>
+<title>Not using latest EAPI</title>
+<body>
+<p>
+Often, user-submitted ebuilds do not use the latest EAPI and may even
+be several versions behind. EAPIs bring new helper functions and improved
+methods for completing common tasks. It's recommended that you port submitted
+ebuilds to the latest EAPI possible (subject to eclass constraints) to improve
```

Comment:
Subject to eclass and base-system (https://wiki.gentoo.org/wiki/Project:Base#Rules_and_limitations) constraints?

Reply:
Please do not include it into "a common mistakes". It is nit a mistake. A person may use whatever EAPI they want as long as it is not deprecated. Maybe make a note but not in common mistakes.

Reply:
In theory, sure, it's not a mistake. But every single time a user makes a PR and doesn't bump the EAPI, we will usually ask them to.